test

```diff
@@ -184,6 +184,15 @@ function dep_tests {
 	fi
 }
 
+function other_tests {
+	echo "Checking other projects..."
+	projects="./contrib/recipes/ ./tools/..."
+	for path in $projects; do
+		echo "testing $path"
+		go test -v $path
```

Comment:
`go test -i -v`? `go test -v` will try to run tests...

have you tried this against broken code in benchmark/cmd?